backend/src/util/index.ts

```diff
@@ -0,0 +1,7 @@
+'use strict';
```

Comment:
Is there a benefit to having this file as util/index.ts instead of util.ts?

Reply:
Removed